src/backend/utils/page-builder.js

```diff
@@ -0,0 +1,264 @@
+/* eslint-disable class-methods-use-this */
```

Comment:
General comment: I would move out all HTML strings somewhere else - the best would be to do this via PUG because we already have this in deps. You can even reuse Renderer class. 

For instance infoList would be:

```javascript
  addInfoList({ items, columns, offset, title, subtitle }) {
    // next line will go to partials/infoList.pug as a mixin, so its not even needed here
    // const itemsContent = items.map(item => this.getInfoListItem(item))
    const partialContent = new Renderer('partials/infoList', { items, columns, offset, title, subtitle }).render()
    this._pageContent.push(partialContent)
```